src/integrations/sdkinfo.ts

```diff
@@ -37,19 +48,18 @@ export class SdkInfo implements Integration {
       }
 
       event.platform = event.platform || 'javascript';
-      event.sdk = {
-        ...event.sdk,
-        name: SDK_NAME,
-        packages: [
-          ...((event.sdk && event.sdk.packages) || []),
-          ...((this._nativeSdkInfo && [this._nativeSdkInfo]) || []),
-          {
-            name: 'npm:@sentry/capacitor',
-            version: SDK_VERSION,
-          },
-        ],
-        version: SDK_VERSION,
-      };
+      event.sdk = event.sdk || {};
+      event.sdk.name = event.sdk.name || defaultSdkInfo.name;
+      event.sdk.version = event.sdk.version || defaultSdkInfo.version;
```

Comment:
We can use SDK_NAME, SDK_VERSION, and remove `defaultSdkInfo` since it's not used anywhere else.